docs/reference/how-to/search-speed.asciidoc

```diff
@@ -326,3 +326,44 @@ queries, they should be mapped as a `keyword`.
 <<index-modules-index-sorting,Index sorting>> can be useful in order to make
 conjunctions faster at the cost of slightly slower indexing. Read more about it
 in the <<index-modules-index-sorting-conjunctions,index sorting documentation>>.
+
+[float]
+=== Use `preference` to optimize cache utilization
+
+There are multiple caches that can help with search performance, such as the
+filesystem cache, the <<shard-request-cache,request cache>> or the
+<<query-cache,query cache>>. Yet all these caches are maintained at the node
+level, meaning that if you run the same request twice in a row, have 1
+<<glossary-replica-shard,replica>> or more and use the default routing
```

Comment:
I'd s/use the default routing algorithm, which is round-robin,/use round-robin, the default routing algorithm/